### DISCUSSION TITLE
Kloudsec 挂了，GitHub Pages 去 HTTPS 化 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#225] [Announcements]
https://mazhuang.org/2016/07/11/remove-https/

**Comment by QingyiZhang** [comment]
我看现在您的首页是https的，我fork了您的主页，做了修改后，开启https，但是显示不正常（感觉css没有加载成功），求问可能原因及如何解决？

**Comment by QingyiZhang** [comment]
我加载了不安全的js后，界面显示正常，但是浏览器https前面显示红色的，提示不安全~

**Comment by QingyiZhang** [comment]
This page is trying to load scripts from unathentication sources

**Comment by mzlogin** [comment]
看现象应该是 _config.yml 里的 url 没配置为 https 的，导致资源还是以 http 的链接在加载，这会被浏览器认为不安全而拒绝。

**Comment by QingyiZhang** [comment]
果然是，已修正，感谢~这篇博文可以修改一下了，目前github page已支持https

**Comment by mzlogin** [comment]
> @QingyiZhang
> 果然是，已修正，感谢~这篇博文可以修改一下了，目前github page已支持https

嗯嗯~已经在文章顶部加了说明和链接。
